conda/meta.yaml

```diff
@@ -40,5 +40,5 @@ requirements:
     - setuptools
     - setuptools_scm
     - ts-salobj
-    - ts-idl
+    - ts-xml
```

Comment:
Do you need to use the following as well?

```
- ts-xml {{ xml_version }}
```

Reply:
We only need to test with a specific version and then allow the run time to install with any version of XML. I believe this is because XML is controlled tightly by the cycle build process and so we can safely assume that it will work in the run time.